learning_journal/learning_journal/__init__.py

```diff
@@ -0,0 +1,14 @@
+from pyramid.config import Configurator
+
+
+def main(global_config, **settings):
+    """ This function returns a Pyramid WSGI application.
+    """
+    config = Configurator(settings=settings)
+    config.include('pyramid_jinja2')
+    config.include('.routes')
+    config.add_static_view('static', 'static', cache_max_age=3600)
+    config.add_route('home', '/')
```

Comment:
Lines 10 and 11 are accounted for in your `routes.py` file. They don't need to be here.